pdudaemon/pdurunner.py

```diff
@@ -53,7 +53,8 @@ def do_job(self, port, request):
         retries = self.retries
         while retries > 0:
             try:
-                return self.driver.handle(request, port)
+                self.driver.handle(request, port)
+                return True
```

Comment:
your commit message claims `Rework do_job to return a boolean such that False means the job succeeded` -- That doesn't seem to match this code?

Reply:
Done.